include/runtime/instance/memory.h

```diff
@@ -35,8 +36,15 @@ class MemoryInstance {
   MemoryInstance() = delete;
   MemoryInstance(const AST::Limit &Lim)
       : HasMaxPage(Lim.hasMax()), MinPage(Lim.getMin()), MaxPage(Lim.getMax()),
-        CurrPage(Lim.getMin()), Data(CurrPage * kPageSize) {}
-  virtual ~MemoryInstance() noexcept = default;
+        CurrPage(Lim.getMin()) {
+    uint8_t *const Pages = static_cast<uint8_t *>(
+        mmap(nullptr, UINT64_C(0x200000000), PROT_READ | PROT_WRITE,
```

Comment:
Should we have a constant value to represent `0x200000000`?

Reply:
Done.